### PR TITLE
Convert a few accessors from Cert::Status to bool/void.

### PR DESCRIPTION
**Pull request by pphaneuf**
This change is dangerous, but I've tested it on top of #916 (with which it is safe). Rebased on top of master, since it's actually independent (but I'd understand if you wouldn't want to merge this before #916!).

### CODE REVIEW

File: cpp/log/cert.cc [hunk 1]
```diff
@@ -1228,33 +1228,32 @@ CertChain::CertChain(const string& pem_string) {
...
   chain_.push_back(cert);
-  return Cert::TRUE;
+  return true;
 }
 
 
-Cert::Status CertChain::RemoveCert() {
-  if (!IsLoaded()) {
+void CertChain::RemoveCert() {
+  if (IsLoaded()) {
```
Comment (Martin2112) [inline cpp/log/cert.cc]: We're losing the status, is it never checked by callers?

File: cpp/log/cert.h [hunk 2]
```diff
@@ -285,21 +285,21 @@ class CertChain {
...
+  // Else returns false.
+  bool AddCert(Cert* cert);
 
   // Remove a cert from the end of the chain.
-  // If successful, returns TRUE.
-  // If the chain is empty, returns ERROR.
-  Cert::Status RemoveCert();
+  // If successful, returns true.
+  // If the chain is empty, returns false.
+  void RemoveCert();
```
Comment (Martin2112) [inline cpp/log/cert.h]: Comment doesn't match void function.

File: cpp/log/cert.cc (hunk 1 above)
Comment (pphaneuf) [inline cpp/log/cert.cc]: Nope!

File: cpp/log/cert.h (hunk 2 above)
Comment (pphaneuf) [inline cpp/log/cert.h]: Updated the comment.

**Pull request merged**